HISTORY.rst

```diff
@@ -42,8 +42,12 @@ unreleased
   models, twins and relationships to a Digital Twins instance. The bulk import data must be stored as a blob in
   a user owned storage account and container.
 
+* Updated `az dt model create` command to process input ontologies larger than 250 models in a single command run.
```

Comment:
0.15.0 was released last week, these changes should go under the `unreleased` section.